Gemfile.lock

```diff
@@ -1,51 +1,49 @@
 PATH
   remote: .
   specs:
-    moesif_rack (1.4.17)
-      moesif_api (~> 1.2.14)
+    moesif_rack (1.5.1)
+      moesif_api (~> 1.2.17)
 
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.8.0)
-      public_suffix (>= 2.0.2, < 5.0)
+    addressable (2.8.4)
+      public_suffix (>= 2.0.2, < 6.0)
     domain_name (0.5.20190701)
       unf (>= 0.0.5, < 1.0.0)
     http-accept (1.7.0)
-    http-cookie (1.0.4)
+    http-cookie (1.0.5)
       domain_name (~> 0.5)
-    json (2.6.1)
+    json (2.6.3)
     json_mapper (0.2.1)
       json (>= 1.4.3)
     mime-types (3.4.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2022.0105)
-    moesif_api (1.2.14)
+    mime-types-data (3.2023.0218.1)
+    moesif_api (1.2.17)
       json_mapper (~> 0.2, >= 0.2.1)
       moesif_unirest (~> 1.1.6)
     moesif_unirest (1.1.6)
       addressable (~> 2.8, >= 2.8.0)
       json (~> 2.6, >= 2.6.1)
       rest-client (~> 2.1, >= 2.1.0)
     netrc (0.11.0)
-    power_assert (2.0.1)
-    public_suffix (4.0.6)
+    power_assert (2.0.3)
+    public_suffix (5.0.1)
     rake (13.0.6)
     rest-client (2.1.0)
       http-accept (>= 1.7.0, < 2.0)
       http-cookie (>= 1.0.2, < 2.0)
       mime-types (>= 1.16, < 4.0)
       netrc (~> 0.8)
-    test-unit (3.5.3)
+    test-unit (3.6.1)
       power_assert
     unf (0.1.4)
       unf_ext
-    unf_ext (0.0.8)
+    unf_ext (0.0.8.2)
 
 PLATFORMS
   arm64-darwin-21
-  ruby
-  x86_64-linux
```

Comment:
@xinghengwang any reason for removing these 2 platforms?

Reply:
not sure. seems auto generated. let me double check,